admin_test.go

```diff
@@ -332,6 +332,139 @@ func TestClusterAdminCreatePartitionsWithoutAuthorization(t *testing.T) {
 	}
 }
 
+func TestClusterAdminAlterPartitionReassignments(t *testing.T) {
+	seedBroker := NewMockBroker(t, 1)
+	defer seedBroker.Close()
+
+	seedBroker.SetHandlerByMap(map[string]MockResponse{
+		"MetadataRequest": NewMockMetadataResponse(t).
+			SetController(seedBroker.BrokerID()).
+			SetBroker(seedBroker.Addr(), seedBroker.BrokerID()),
```

Comment:
Please could you add a second mockBroker to the test and set that as the controller in the mock metadata response and _not_ the seed broker? That should exercise that the request must be sent to the Controller and not just any broker (or the broker currently connected to)